### PR TITLE
[CloudPC]Support AADJ for OPNC

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

**Comment by RuiHou105** [comment]
**Roadmap workitem:** 

Feature 11399: [Devices and apps][Cloud PC] Support AADJ for OPNC

File: api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "cloudPcOnPremisesConnection: updateAdDomainPassword"
-description: "Update AD domain password for a successful onPremisesConnection."
+description: "Update the Active Directory (AD) domain password for a successful on-premises connection. This API is supported when the onPremisesConnection's type is hybridAzureADJoin."
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md:3]: Do you mean Active Directory here or do you mean Azure Active Directory (Azure AD)? Active Directory is a legacy term.

**Review by AshleyYangSZ — APPROVED** [review]

File: api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md:3]: Updated. Please check.

Comment (JarbasHorst) [reply api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md]: Verified @Lauragra ✅

**Comment by RuiHou105** [comment]
Hi @Lauragra , all comments are resolved. As the schema is already alive, could you please take a second look when you got time? Thanks a lot!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**